sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/AzureStorageCheckpointLeaseManager.cs

```diff
@@ -348,18 +352,29 @@ async Task<bool> AcquireLeaseCoreAsync(AzureBlobLease lease)
                     }
 
                     ProcessorEventSource.Log.AzureStorageManagerInfo(this.host.HostName, lease.PartitionId, "Need to ChangeLease");
-                    renewLease = true;
-                    newToken = await leaseBlob.ChangeLeaseAsync(
-                        newLeaseId,
-                        AccessCondition.GenerateLeaseCondition(lease.Token),
-                        null,
-                        this.operationContext).ConfigureAwait(false);
+                    if (IsInfiniteLease)
+                    {
+                        //Incase of inifinite lease. we need to break the lease and acquire it again.
+                        TimeSpan? breakReleaseTime = TimeSpan.FromSeconds(3);
```

Comment:
the local variable should not be nullable. You can pass non-nullable value to a nullable parameter.